### DISCUSSION TITLE
useTransform outside of React context, is it possible?

### DISCUSSION THREAD

**Discussion by jmrhoades** [motiondivision/motion#2355] [Q&A]
I'm creating motion values like this:
```
const x = motionValue(100);
const zoom = motionValue(1);
```
And I'd like to chain them together with a transform like this:
```
const zoomedX = useTransform(() => x.get() * zoom.get());
```
But `useTransform` must be used inside a React component and the `transform` method expects an array of values, not a function that listens to motion value changes.

Is there a way to do a transform like this without hooks?

**Comment by BraveLilToaster** [comment]
Framer includes universal packages which can be used outside of react or without react if import from `"framer-motion/dom"`. It is  recommended to use the react hooks if you are using react but it looks like you are not using react so I would give it a shot. Hope that is helpful :smile: 

https://www.framer.com/motion/transform-function/
